app/controllers/users_controller.rb

```diff
@@ -1,4 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
-  def show; end
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+    @posts = @user.posts.limit(3)
```

Comment:
- [ ] Kindly use your created `three_most_recent_posts` method which was created in the previous milestone so kindly fix this issue :slightly_smiling_face: 

```suggestion
    @posts = @user.three_most_recent_posts
```